src/components/SkillsToDo/SkillsToDo.js

```diff
@@ -1,28 +1,17 @@
-import React, { useState } from 'react';
-import { Card, Button } from 'react-bootstrap';
+import React from "react";
+import { Card } from "react-bootstrap";
 
-import SkillToDoItem from './SkillToDoItem/SkillToDoItem';
-import Timer from 'components/Timer/Timer';
-
-
-function SkillsToDo({ projects = [], open = true, updatePractisedSkill = () => { } }) {
-  const [showTimer, setShowTimer] = useState(false);
+import SkillToDoItem from "./SkillToDoItem/SkillToDoItem";
 
+function SkillsToDo({ projects = [], open = true, updatePractisedSkill = () => {} }) {
   return (
-    <>
-      {showTimer
-        ? <>
-          <Button onClick={() => setShowTimer(false)}>Close Pomodoro Timer</Button>
-          <Timer duration={1500}/>
-        </>
-        : <Button onClick={() => setShowTimer(true)}>Show Pomodoro Timer</Button>}
```

Comment:
You seem to have deleted the import of the timer. Maybe this happened on merging or was it deliberate? I think this is the only error so I can make the changes to convert this file back this morning if you don't have time

Reply:
I have made these edits and merged in a new branch.